python/py-torchvision/Portfile

```diff
@@ -4,30 +4,28 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
+github.setup        pytorch vision 0.9.1 v
+revision            1
 name                py-torchvision
-version             0.9.1
-revision            0
-github.setup        pytorch vision ${version} v
-
-checksums           rmd160  946000baedbc1cd14b7d8e59587d703449f2aa2c \
-                    sha256  e836d7d7f256972f254b3a65099e25c2bdc76ccd39348fa0c46a083b51561b53 \
-                    size    7219847
 
+license             BSD
+maintainers         nomaintainer
 platforms           darwin
 supported_archs     x86_64
 
-license             BSD
+description         PyTorch datasets, transforms and models specific to \
+                    computer vision
 
-maintainers         nomaintainer
+long_description    The torchvision package consists of popular \
+                    datasets, model architectures, and common image \
+                    transformations for computer vision.
 
-description         PyTorch datasets, transforms and models specific to computer vision
-long_description    {*}${description}
+checksums           rmd160  946000baedbc1cd14b7d8e59587d703449f2aa2c \
+                    sha256  e836d7d7f256972f254b3a65099e25c2bdc76ccd39348fa0c46a083b51561b53 \
+                    size    7219847
 
-# Support python versions
 python.versions     37 38 39
 
-compiler.cxx_standard 2011
```

Comment:
Why remove ? Are you sure the build doesn’t need this, it must have been added for a reason.

if you check the build logs you will find the builds actually use ` -std=c++14` so please add this back and up the standard accordingly.

Reply:
Thanks done.